Sources/TuistSupport/Utils/Environment.swift

```diff
@@ -45,6 +45,24 @@ public protocol Environmenting: AnyObject {
 
 /// Local environment controller.
 public class Environment: Environmenting {
+    
+    private enum EnvFileFailure: Error, CustomDebugStringConvertible {
+        case malformedKeyValuePair
+        case contentsUnreadable
+        case fileMissing(path: String)
+        
+        fileprivate var debugDescription: String {
+            switch self {
+            case .malformedKeyValuePair:
+                return "key value pair was malformed: make sure they follow the format `KEY=VALUE`"
```

Comment:
The descriptions here will be output to the user when it errors, so it's important that they are clear and actionable. For example, in `.malformedKeyValuePair` it's useful to include the path to the file where the issue is.